pkg/resourceinterpreter/default/thirdparty/resourcecustomizations/apps.kruise.io/v1beta1/StatefulSet/customizations.yaml

```diff
@@ -0,0 +1,121 @@
+            if statusItems[i].status ~= nil and statusItems[i].status.readyReplicas ~= nil then
+              readyReplicas = readyReplicas + statusItems[i].status.readyReplicas
+            end
+            if statusItems[i].status ~= nil and statusItems[i].status.currentReplicas ~= nil then
+              currentReplicas = currentReplicas + statusItems[i].status.currentReplicas
+            end
+            if statusItems[i].status ~= nil and statusItems[i].status.updatedReplicas ~= nil then
+              updatedReplicas = updatedReplicas + statusItems[i].status.updatedReplicas
+            end
+            if statusItems[i].status ~= nil and statusItems[i].status.availableReplicas ~= nil then
+              availableReplicas = availableReplicas + statusItems[i].status.availableReplicas
+            end
+            if statusItems[i].status ~= nil and statusItems[i].status.updatedReadyReplicas ~= nil then
+              updatedReadyReplicas = updatedReadyReplicas + statusItems[i].status.updatedReadyReplicas
+            end            
+            if statusItems[i].status ~= nil and statusItems[i].status.updateRevision ~= nil and statusItems[i].status.updateRevision ~= '' then
+              updateRevision = statusItems[i].status.updateRevision
+            end
+            if statusItems[i].status ~= nil and statusItems[i].status.currentRevision ~= nil and statusItems[i].status.currentRevision ~= '' then
+              currentRevision = statusItems[i].status.currentRevision
+            end
+          end
+          desiredObj.status.observedGeneration = generation
+          desiredObj.status.replicas = replicas
+          desiredObj.status.readyReplicas = readyReplicas
+          desiredObj.status.currentReplicas = currentReplicas
+          desiredObj.status.updatedReplicas = updatedReplicas
+          desiredObj.status.availableReplicas = availableReplicas
+          desiredObj.status.updatedReadyReplicas = updatedReadyReplicas
+          desiredObj.status.updateRevision = updateRevision
+          desiredObj.status.currentRevision = currentRevision
+          return desiredObj
+        end
+    statusReflection:
+      luaScript: >
+        function ReflectStatus (observedObj)
+          status = {}
+          if observedObj == nil or observedObj.status == nil then
+            return status
+          end
+          status.replicas = observedObj.status.replicas
+          status.readyReplicas = observedObj.status.readyReplicas
+          status.currentReplicas = observedObj.status.currentReplicas
+          status.updatedReplicas = observedObj.status.updatedReplicas
+          status.availableReplicas = observedObj.status.availableReplicas
+          status.updateRevision = observedObj.status.updateRevision
+          status.currentRevision = observedObj.status.currentRevision
+          status.updatedReadyReplicas = observedObj.status.updatedReadyReplicas
+          return status
+        end
+    healthInterpretation:
```

Comment:
`apps.kruise.io` statefulset is a reference to the code in `pkg/resourceinterpreter/default/native/healthy.go` for the health part of native statefulset. Compared to cloneset, statefulset health is more relaxed, the idea is changed from "determine what is healthy" to "determine what is unhealthy/illegal". It allows a member cluster to be considered healthy when the pod is not fully ready, and to be aggregated.